### ISSUE TITLE
Scanning URL gives Permission denied to access document

### ISSUE THREAD

**Issue by devanshah2** [IBMa/equal-access#5]
##### APPLIES TO:
(Tool(s) that this defect affects, in the case of rule defect please include the rule name)
@ibma/karma-ibma

##### PROBLEM DESCRIPTION:
(Describe the problem you are seeing in as much detail as you can)

When scanning a URL which is not in the same domain/port/protocol as the karma server it throws exception "permission denied to access property "document""

##### STEPS TO REPRODUCE:
(Steps to reproduce the issues from start to finish.)
(Include a sample html file when possible (attach under the "Links" section), Include a link if sample file cannot be produced.)
	1. Scan https://internetofthings.ibmcloud.com/#/
	2. notice that there will be an exception throw 

##### EXPECTED OUTPUT:
(The expected results that you should see)

No exception, should match baseline

##### ACTUAL OUTPUT:
(The current results that you are seeing) (Include Description and/or a screenshot of what you are seeing)

permission denied to access property "document"
